crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -707,6 +689,12 @@ impl Room {
             .get_event_room_receipt_events(self.room_id(), receipt_type, thread, event_id)
             .await
     }
+
+    /// Returns an active (on-going) room call (also known as matrix RTC
+    /// session).
+    pub fn active_room_call(&self) -> Option<RoomCall> {
```

Comment:
here `active_room_call` also makes a lot of sense since this is filtering for application = call and call_id = ""